frontend/src/pages/HaroldAISettings/HaroldAISettings.tsx

```diff
@@ -42,14 +42,14 @@ const AI_FEATURES: AiSetting[] = [
 ]
 
 export const HaroldAISettings = () => {
-	const { workspace_id } = useParams<{ workspace_id: string }>()
+	const { currentWorkspace } = useApplicationContext()
```

Comment:
Tangential change - Update Harold to use the same context as other places using `useGetWorkspaceSettingsQuery`